recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
     sha256: {{ sha256 }}
 
 build:
-    number: 0
+    number: 1
```

Comment:
Yeah, re-rendering doesn't normally get a build number bump. We have had to do build number bumps sometimes with re-renders if there are changes to the recipe or if we actually need a new package deployed.